include/mdw_dslash5_tensor_core.cuh

```diff
@@ -10,12 +10,11 @@
 
 #include <cub_helper.cuh>
 
-#if (__CUDACC_VER_MAJOR__ >= 9 && __COMPUTE_CAPABILITY__ >= 700)
+#if (CUDA_VERSION >= 9000 && __COMPUTE_CAPABILITY__ >= 700)
```

Comment:
@hummingtree FYI, not sure if this is reconciled in your branch yet

Reply:
Note the need for this is for compatibility with clang: it's not a CUDA compiler so doesn't identify as such.  The cross compiler mechanism is to use `CUDA_VERSION` instead.